docs/sink/flink-connector-obkv-hbase.md

```diff
@@ -113,6 +115,49 @@ public class Main {
 }
 ```
 
+##### odp mode
+
+```java
+package com.oceanbase;
+
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        StreamExecutionEnvironment env =
+                StreamExecutionEnvironment.getExecutionEnvironment().setParallelism(1);
+        StreamTableEnvironment tEnv =
+                StreamTableEnvironment.create(
+                        env, EnvironmentSettings.newInstance().inStreamingMode().build());
+
+        tEnv.executeSql(
+                "CREATE TABLE t_sink ( "
+                        + " rowkey STRING,"
+                        + " family1 ROW<column1 STRING, column2 STRING>,"
+                        + " PRIMARY KEY (rowkey) NOT ENFORCED"
+                        + ") with ("
+                        + "  'connector'='obkv-hbase',"
+                        + "  'odp-mode'='true',"
+                        + "  'odp-ip'='127.0.0.1',"
+                        + "  'odp-port'='12881',"
```

Comment:
It's better to take the default rpc '2885' port for example.

Reply:
updated 2885